scss/modules/modals/_channel.scss

```diff
@@ -107,3 +107,19 @@
 .notification_prefs_icon::before {
   color: $base-font-color;
 }
+
+.channel_header {
+  .channel_header__tabs {
+    .channel_header__tab {
+      color: $base-link-color;
+
+      &.--active {
```

Comment:
I believe you may have meant for this to be `&--active`? With the period in there after the ampersand it attempts to select on the class `--active` which I don't think Slack uses. Also I'm not sure it's legal css to start a class name with two hyphens.

Reply:
![image](https://user-images.githubusercontent.com/2967322/53751886-4ae60980-3e7b-11e9-8551-b6ef9edb27de.png)
This is the "Apps" section header which does have the `--active` as the class name

Reply:
Well TIL :)

Reply:
😄